network-api/networkapi/templates/tags/multipage_menu_link_note.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+
+{% with class="tw-text-red-80 tw-text-sm tw-ml-1" note_for_translators="This note is internal facing only and is only visible to our CMS logged in users." %}
+    {% comment %}These notes are internal facing only. Only logged in users can see them.{% endcomment %}
+
+    {% if menu_entry.restriction == "login" %}
+        <span class="{{ class }}">{% trans "(logged in users only)" context note_for_translators %}</span>
+    {% elif menu_entry.restriction == "groups" %}
+        <span class="{{ class }}">{% trans "(restricted to specific groups)" context note_for_translators %}</span>
+    {% elif menu_entry.restriction == "password" %}
+        <span class="{{ class }}">{% trans "(password protected)" context note_for_translators %}</span>
+    {% endif %}
+
+    {% if not menu_entry.page.live %}
+        <span class="{{ class }}">{% trans "(draft)" context note_for_translators %}</span>
+    {% endif %}
```

Comment:
Hey @mmmavis, sorry for hijacking this PR (stumbled upon this), but can we please hardcode these strings? We shouldn’t translate internal facing content.
Also the note doesn’t get properly exported to the .po files

Thanks!

Reply:
Thanks @TheoChevalier ! Initially I didn't wrap them in trans tags for the same reason. I'll revert my commit.